README.md

```diff
@@ -7,11 +7,13 @@ This git repository contains PoC of improving Foreman documentation.
 ## Foreman Guides
 
 This is a tree of documentation based on Red Hat Satellite 6 official books.
-Contributions are welcome.
 See README in the guides/ subdirectory for more information.
 
 * [guides](guides/README.md)
 
+Contributions are welcome.
+New contributions are subjected to technical review for accuracy and editorial review for consistency. For an overview of what to expect from editorial review, see [Peer review guide for technical documentation](https://redhat-documentation.github.io/peer-review/#checklist).
```

Comment:
I'd consider moving this up to the main section or even introducing a `Contributing` section.